polaris-react/src/components/Badge/components/Pip/Pip.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import {classNames, variationName} from '../../../../utilities/css';
+import type {Progress, Status} from '../../types';
+import {VisuallyHidden} from '../../../VisuallyHidden';
+
+import styles from './Pip.scss';
+
+export interface PipProps {
+  status?: Status;
+  progress?: Progress;
+  accessibilityLabel: string;
```

Comment:
I’m still not convinced about the accessibility label being required when in most cases a good default can be generated from the `status` and `progress` props.